conf.d/haproxy.yaml.example

```diff
@@ -5,6 +5,10 @@ instances:
     # username: username
     # password: password
     #
+    # or, with a unix stats or admin socket:
+    #
+    # url: unix:///var/run/haproxy.sock
```

Comment:
Yeah at first I thought it'd be clearer but doesn't seem so...
Feel free to adjust it as you think it's better from the user perspective, or leave it as it is and we'll fix that later.